geoips/create_plugin_registries.py

```diff
@@ -12,6 +12,7 @@
 created at the top level package directory for each plugin package.
 """
 
+import warnings
```

Comment:
@biosafetylvl5 Look at getting the warnings module output streamed into logging.warning logging level

Reply:
Done